src/ui/Modal/QuestionModal.js

```diff
@@ -61,14 +62,16 @@ const QuestionModal = ({
       <Button variant={ButtonVariants.SECONDARY} onClick={onClose}>
         {cancelTitle}
       </Button>
-      <Button
-        loading={confirmLoading}
-        variant={ButtonVariants.PRIMARY}
-        onClick={onConfirm}
-        disabled={confirmButtonDisabled || confirmLoading}
-      >
-        {confirmTitle}
-      </Button>
+      {!confirmButtonHidden && (
```

Comment:
```suggestion
      {confirmButtonVisible && (
```

nit.
Most of the time you want the positive as a prop instead. You can make the default true